src/Fantomas/CodePrinter.fs

```diff
@@ -913,12 +913,17 @@ and genAnonRecordFieldName astContext (AnonRecordFieldName (s, e)) =
 
     !-s +> sepEq +> expr
 
-and genTuple astContext es =
+and genTuple astContext es isBaseCtorCall =
```

Comment:
Don't change `genTuple` here, your edge case is in `MDImplicitInherit`. It should be tackle in full over there.